src/encoded/audit/file.py

```diff
@@ -7,33 +7,42 @@
 )
 from .standards_data import pipelines_with_read_depth
```

Comment:
add #no qa to the end of this line to prevent import warning

Reply:
This import was unused - so I removed it